core/examples/luigi-sample-angular/src/app/overview/overview.component.ts

```diff
@@ -57,4 +57,11 @@ export class OverviewComponent {
       description: 'navigation node configuration to redirect to another path'
     }
   ];
+
+  private sendDirtyEvent = () => {
```

Comment:
It would be nice to have a dedicated method for setting dirty/not dirty state in luigi client - unless we decided to skip it for this issue.

Reply:
Good idea, I'll create it